asserts/model_test.go

```diff
@@ -536,6 +556,9 @@ func (mods *modelSuite) TestDecodeInvalid(c *C) {
 		{serialAuths, "serial-authority:\n  - 5_6\n", `"serial-authority" header must be a list of account ids`},
 		{sysUserAuths, "system-user-authority:\n  a: 1\n", `"system-user-authority" header must be '\*' or a list of account ids`},
 		{sysUserAuths, "system-user-authority:\n  - 5_6\n", `"system-user-authority" header must be '\*' or a list of account ids`},
+		{preseedAuths, "preseed-authority:\n  a: 1\n", `"preseed-authority" header must be a list of account ids`},
+		{preseedAuths, "preseed-authority:\n  - 5_6\n", `"preseed-authority" header must be a list of account ids`},
+		{preseedAuths, "preseed-authority: *\n", `"preseed-authority" header must be a list of account ids`},
```

Comment:
(nitpick) while looking at this I noticed we seem to lack a similar check for serial-authoriy about "*", so
```go
+               {serialAuths, "serial-authority: *\n", `"serial-authority" header must be a list of account ids`},
```
could be added above (but strictly speaking it could be it's own followup as it's not really related to this PR :)